transport/graphsync/receiver.go

```diff
@@ -56,50 +59,39 @@ func (r *receiver) receiveRequest(ctx context.Context, initiator peer.ID, incomi
 		ch = r.transport.trackDTChannel(chid)
 	}
 
-	if receiveErr == datatransfer.ErrResume && ch.Paused() {
-
-		var extensions []graphsync.ExtensionData
+	if receiveErr != nil {
 		if response != nil {
```

Comment:
What's happening in here now if `response == nil`, is it right that we continue on (i.e. that this is really `if receiveErr != nil && response != nil {`. It looks like we can get all the way down to the end and do a `r.transport.processAction()` with a `nil` `response`.